ansible/roles/prometheus/templates/prometheus.yml

```diff
@@ -214,4 +214,38 @@ scrape_configs:
     static_configs:
       - targets:
         - backend-hel.ooni.org:444
+
+  # EC2 instances monitoring: 
+  - job_name: 'ooni-api-ec2'
```

Comment:
Maybe the name of this job should be `ooni-aws-ec2`? I suppose we can apply this to anything we would run on ec2 not necessarily just the API?

Reply:
Yeah that's true! I used that name at the beginning because it was intended for the API stuff but then we changed our minds and `ooni-aws-ec2` makes more sense now